modules/nix-bitcoin.nix

```diff
@@ -1,9 +1,44 @@
-# This file exists only for backwards compatibility
+{ config, pkgs, lib, ... }:
 
-{ lib, ... }:
+with lib;
 {
-   imports = [
-     ./presets/secure-node.nix
```

Comment:
We need to change `examples/minimal-configuration.nix` to import `modules.nix` instead of `nix-bitcoin.nix`. Fwiw the minimal container was buggy even before your commit because it imported `secure-node.nix` instead of being minimal.

Reply:
Thanks, fixed!